mempool/clist_mempool.go

```diff
@@ -720,12 +744,23 @@ func (mem *CListMempool) Update(
 	}
 
 	// Update metrics
-	mem.metrics.Size.Set(float64(mem.Size()))
-	mem.metrics.SizeBytes.Set(float64(mem.SizeBytes()))
+	for lane := range mem.lanes {
+		mem.updateSizeMetrics(lane)
+	}
 
 	return nil
 }
 
+// updateSizeMetrics updates the size-related metrics of a given lane.
+func (mem *CListMempool) updateSizeMetrics(lane types.Lane) {
+	label := strconv.FormatUint(uint64(lane), 10)
+	mem.metrics.LaneSize.With("lane", label).Set(float64(mem.lanes[lane].Len()))
+	mem.metrics.LaneBytes.With("lane", label).Set(float64(mem.LaneBytes(lane)))
+	// TODO: do we want to keep the following redundant metrics? The total sizes can be computed from the other two.
+	mem.metrics.Size.Set(float64(mem.Size()))
+	mem.metrics.SizeBytes.Set(float64(mem.SizeBytes()))
```

Comment:
Do we want to keep these two redundant metrics? The total sizes can be computed from the lane sizes.

Reply:
Can you easily do it in prometheus or grafana? If yes, then we could mark it as deprecated because this metric is heavily used, I would not just remove it.

Reply:
Yes, in Prometheus for example `cometbft_mempool_size` is the same as `sum(cometbft_mempool_lane_size) by (job)`. I will mark them as deprecated.

Reply:
Cool, then I think we can mark it as deprecated?